run.sh

```diff
@@ -59,15 +59,13 @@ java_() {
 gz() {
 	fifo="$TMP/$(uuidgen)-$(basename "$1")"
 	mkfifo "$fifo"
-	echo "$fifo"
```

Comment:
I thought this line was crucial to get the name of the fifo as output of this function, so the `> "$(gz ...)"` construct works. Same for the `guz`.